internal/mesh/exports.go

```diff
@@ -28,33 +30,49 @@ var (
 	TCPRouteKind               = types.TCPRouteKind
 	DestinationPolicyKind      = types.DestinationPolicyKind
 	ComputedRoutesKind         = types.ComputedRoutesKind
+	ProxyStateTemplateKind     = types.ProxyStateTemplateKind
 
 	// Resource Types for the v1alpha1 version.
 
 	ProxyConfigurationV1Alpha1Type              = types.ProxyConfigurationV1Alpha1Type
 	UpstreamsV1Alpha1Type                       = types.UpstreamsV1Alpha1Type
 	UpstreamsConfigurationV1Alpha1Type          = types.UpstreamsConfigurationV1Alpha1Type
-	ProxyStateTemplateConfigurationV1Alpha1Type = types.ProxyStateTemplateV1Alpha1Type
+	ProxyStateTemplateV1Alpha1Type              = types.ProxyStateTemplateV1Alpha1Type // TODO(rb): which one is it?
```

Comment:
@ishustava when i integrated this with `main` i noticed there were two names in play. I don't know which one ultimately is supposed to be used so i ensured both variations were kept here as interchangeable temporarily.

Reply:
`ProxyStateTemplateConfigurationV1Alpha1Type` is a typo from an earlier PR.